### PR TITLE
adapt changelog

### PR DESCRIPTION
**Pull request by DrAnaximandre**
Modifies the changelog to take into account new changes before 0.3.0 bump.

---
This change is [Reviewable](https://reviewable.io/reviews/tboquet/python-alp/56)

### CODE REVIEW

**Comment by tboquet** [comment]
@DrAnaximandre are we good to go?

**Comment by DrAnaximandre** [comment]
let me double check since there have been some modifs on the CLI

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Changelog
 =========
 
 
-Unreleased
+0.2.0 (2017-01-17)
```
Comment (tboquet) [inline CHANGELOG.rst]: 0.3.0

**Review by tboquet — APPROVED** [review]

**Comment by tboquet** [comment]
Reviewed 1 of 1 files at r4, 1 of 1 files at r5.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/tboquet/python-alp/56)*

**Pull request merged**